### PR TITLE
Add state property to LavalinkNode

### PR DESCRIPTION
**Pull request by Skye-31**

### CODE REVIEW

File: src/lib/LavalinkNode.ts [hunk 1]
```diff
@@ -44,6 +44,10 @@ export class LavalinkNode {
      * The resume timeout
      */
     public resumeTimeout = 120;
+    /**
+     * Extra info attached to your node, not required and is not sent to lavalink, purely for you.
+     */
+    public state?: object;
```
Comment (MrJacz) [inline src/lib/LavalinkNode.ts]: make this any. While i dont agree with any. state can be anything

File: src/lib/Types.ts [hunk 2]
```diff
@@ -285,6 +285,10 @@ export interface LavalinkNodeOptions {
      * Resume timeout
      */
     resumeTimeout?: number;
+    /**
+     * Extra info attached to your node, not required and is not sent to lavalink, purely for you.
+     */
+    state?: object;
```
Comment (MrJacz) [inline src/lib/Types.ts]: make this any.

**Review by MrJacz — CHANGES_REQUESTED** [review]

File: src/lib/LavalinkNode.ts (hunk 1 above)
Comment (Skye-31) [reply src/lib/LavalinkNode.ts]: updated

File: src/lib/Types.ts (hunk 2 above)
Comment (Skye-31) [reply src/lib/Types.ts]: updated

**Review by MrJacz — APPROVED** [review]
LGTM

**Pull request merged**